source/platforms/github/GitHubAPI.ts

```diff
@@ -130,6 +130,11 @@ export class GitHubAPI {
       return parseInt(perilID)
     }
 
+    const useGitHubActionsID = process.env["GITHUB_WORKFLOW"]
```

Comment:
After https://github.com/danger/danger-js/pull/935/commits/27da052541110cae73bb6f897dce0f3160aa69d6 
(on 9.2.3) github actions doesn't actually replace its messages.
This is because when you actually send the user request on action fails with

```
Request failed [403]: https://api.github.com/user
Response: {
 "message": "Resource not accessible by integration",
 "documentation_url": "https://developer.github.com/v3/users/#get-the-authenticated-user"
}
```

Reply:
This changes was tested and now it works again